src/test/java/model/LottoCompareResultTest.java

```diff
@@ -0,0 +1,71 @@
+package model;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class LottoCompareResultTest {
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, 0, 1, 1, 2, 2})
```

Comment:
`@ValueSource`에 들어간 값은 한 테스트에 하나씩 사용되는 것으로 알고 있습니다! 
그리고, `@ParameterizedTest`를 사용할 경우에도 이름을 예쁘게 출력해 줄 수 있는데요!
나노 팁 하나 드리고 갑니다 !!!
```suggestion
    @ParameterizedTest(name = "동일한 숫자 개수가 {0}개일 때 Optional.empty()가 반환되어야 한다.")
    @ValueSource(ints = {0, 1, 2})
```
![image](https://github.com/woowacourse-precourse/java-lotto-6/assets/76612738/0fe73379-2cfc-4505-947d-42ac7455b765)

Reply:
그렇군요 테스트 코드가 익숙하지 않아서 좀 더 공부가 필요하겠네요 감사합니다!